src/ui/controls/color-picker/hsb-picker.tsx

```diff
@@ -118,15 +121,24 @@ export default function HSBPicker(props: HSBPickerProps) {
     }
 
     function onSBCanvasRender(canvas: HTMLCanvasElement) {
+        if (isElementHidden(canvas)) {
+            context.store.wasPrevHidden = true;
```

Comment:
Use `store` instead of `context.store` everywhere, there's `const store = context.store` in the beginning. This object reference is always the same for the same virtual node.